presto-orc/src/test/java/io/prestosql/orc/TestingOrcPredicate.java

```diff
@@ -348,6 +371,26 @@ protected boolean chunkMatchesStats(List<String> chunk, ColumnStatistics columnS
                     .map(Slices::utf8Slice)
                     .collect(toList());
 
+            BloomFilter bloomFilter = columnStatistics.getBloomFilter();
+            if (bloomFilter != null) {
+                for (Slice slice : slices) {
+                    if (!bloomFilter.test(slice.getBytes())) {
+                        return false;
+                    }
+                }
+                int falsePositive = 0;
+                byte[] testBuffer = new byte[32];
+                for (int i = 0; i < 100_000; i++) {
+                    ThreadLocalRandom.current().nextBytes(testBuffer);
+                    if (bloomFilter.test(testBuffer)) {
+                        falsePositive++;
+                    }
+                }
+                if (falsePositive != 0 && 1.0 * falsePositive / 100_000_000 > 0.5) {
```

Comment:
This should be `100_000`, too. Maybe pull it into a variable and reuse to avoid future discrepancies.